src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -259,18 +259,34 @@ public int MaxConnectionsPerServer
         
         public long MaxRequestContentBufferSize
         {
-            get { return HttpContent.MaxBufferSize; }
+            // This property has been deprecated. In the .NET Framework it was only used when the handler needed to 
```

Comment:
>Shouldn't we add an ObsoleteAttribute in that case?

Maybe, but I'll open an issue for discussion.  Adding that attribute can break compilation as we have discovered with trying to add obsolete attributes for Ssl2, etc.